Feature/filter mentor

farhadibehnamdev opened the pull request.

unityali99 committed:
Fixed icons size

Hossein-K-R committed:
Update Header.jsx

Updated Header links

alimh17 committed:
fix layout

farhadibehnamdev committed:
✨ feat(ui):Implement filter component.

farhadibehnamdev committed:
Pulled changes from main

farhadibehnamdev committed:
🐛 fix(ui):Fix structure of elements.

unityali99 committed:
Reverted index.js to its previous state and added mini cards to it

ehsangazar committed:
update lock

ehsangazar commented:
<img width="961" alt="Screenshot 2023-07-09 at 8 39 13 am" src="https://github.com/PersiaJS/devmentor-frontend/assets/1703553/d97edd15-ca2a-45f8-a329-91f46e581883">

Failed
Also please don't make Components inside the other folder and stuff, follow the pattern

ehsangazar committed:
fix header

ehsangazar committed:
fix styling homepage

ehsangazar committed:
auth pages

farhadibehnamdev committed:
♻️ refactor:Reorganized component file structure.

This refactor enhances code organization and ensures adherence to the team's file structure conventions.

Fix #17

farhadibehnamdev committed:
🔥 chore:Remove extra layout from containers folder

farhadibehnamdev committed:
♻️ refactor:Rename layout.jsx to index.jsx.

ehsangazar commented:
Fix the conflicts please

farhadibehnamdev committed:
Merge branch 'main' of https://github.com/PersiaJS/devmentor-frontend into feature/filter-mentor
